Add Make help command

twunderlich-grapl opened the pull request:
This adds a help command to Make and lists descriptions for commands

Commit:
Add make help command

This adds a help command to make and lists descriptions for commands

christophermaier commented on Makefile:
```diff
@@ -1,55 +1,74 @@
+	@printf -- '                (≡)    / /_/ // /   / /_/ // /_/ // /   \n'
+	@printf -- '                /      \__, //_/    \__,_// .___//_/    \n'
+	@printf -- '             (≡)      /____/             /_/            \n'
+	@printf -- '\n'
```
Not sure we need the Grapl ASCII art for this, given that it's not the name of the project at hand.

twunderlich-grapl replied:
Yeah I went back and forth about it since it is part of the grapl repo. I think its fair to remove it though

Commit:
Remove Grapl ascii art

While this is a Grapl project, its not really Grapl-related so we can
remove the extra branding
